src/styles/_search_results.scss

```diff
@@ -1,228 +1,100 @@
-      color: $field-text-color;
-    }
-
-    .see-all-filters {
-      background: none;
-      color: $link-color;
-      cursor: pointer;
-      display: block;
-      padding: 10px;
+.search__results {
+  margin: 0 var(--mm-spacing);
+  max-width: 960px;
+  min-height: 500px;
+}
 
-      &[aria-hidden='true'] {
-        display: none;
-      }
+.search__results-header {
+  margin-bottom: calc(var(--mm-spacing) / 2);
 
-      &:hover {
-        text-decoration: underline;
-      }
-
-      &::after {
-        content: ' \2304';
-        font-weight: bold;
-      }
-    }
+  .search__results-heading {
+    color: #00374c;
+    font-family: var(--mm-font-stack-display);
+    font-weight: 600;
+    line-height: 1.2em;
+    margin: 0;
+    padding-top: var(--mm-spacing);
+    position: relative;
+    scroll-margin-top: calc(var(--page-header-height) + var(--mm-spacing));
   }
 
-  &-subheading {
-    font-size: 18px;
-    font-weight: $font-weight-semibold;
+  .search__results-count {
+    font-size: 14px;
   }
 
-  &-list {
-    margin-bottom: 25px;
-
-    > li {
-      padding: 20px 0;
-
-      &:first-child {
-        border-top: 1px solid $border-color;
-      }
-
-      h2 {
-        margin-bottom: 0;
-      }
-    }
+  p {
+    margin: 0;
   }
 
-  .meta-group {
-    align-items: center;
-    clear: both;
-    color: #666;
-    display: block;
-    @include desktop { display: flex; }
-
-    > li {
-      display: block;
-      @include desktop { display: inline; }
-    }
-
-    li:first-child {
-      @include desktop { flex: 1; }
-    }
-
-    .meta-data {
-      color: inherit;
-
-      &:not(:last-child) {
-        [dir='ltr'] & {
-          margin-right: 20px;
-        }
-
-        [dir='rtl'] & {
-          margin-left: 20px;
-        }
-      }
+  a {
+    border-bottom: 2px dotted var(--mm-color-display-text);
+    color: var(--mm-color-display-text);
 
-      &::after {
-        content: none;
-      }
+    &:hover {
+      border-bottom-style: solid;
+      text-decoration: none;
     }
   }
-
-  &-description {
-    margin-top: 10px;
-    word-break: break-word;
-  }
 }
 
-.search-result {
-  &-title {
-    display: inline-block;
-    font-size: $font-size-bigger;
-  }
+.search__result-list-item {
+  margin-bottom: calc(var(--mm-spacing) / 1.5);
 
-  &-icons {
-    [dir='ltr'] & {
-      float: right;
-    }
-
-    [dir='rtl'] & {
-      float: left;
-    }
-  }
-
-  &-votes,
-  &-meta-count {
-    color: $secondary-text-color;
-    display: inline-block;
-    font-size: $font-size-small;
-    padding: 4px 5px;
-    position: relative;
-
-    &-icon {
-      color: $brand-color;
-      height: 13px;
-      vertical-align: middle;
-      width: 13px;
-    }
-
-    [dir='ltr'] & {
-      margin-left: 5px;
-
-      &::before {
-        margin-right: 3px;
-      }
-    }
+  .search__result-title {
+    margin: 0;
 
-    [dir='rtl'] & {
-      margin-right: 5px;
+    a {
+      color: var(--mm-color-logo-blue-light);
```

Comment:
I realize we are using this blue shade of blue on the dev site search results, but the link color does not meet WCAG AAA contrast standards. We should use `--mm-color-display-text` instead. I'll get this updated on the dev site as well.